docs/tutorials/hosting/modrinth.md

```diff
@@ -0,0 +1,11 @@
+# Publishing to Modrinth
+
+Exporting a Modrinth pack is as simple as running `packwiz modrinth export` - this gives you a `.mrpack` in your pack folder that you can upload to Modrinth!
+
+Since this pack format doesn't support side-only mods, packwiz can't create a pack that differs between server and client, and will export a pack for Minecraft clients (containing mods with side `client` or `both`). Mods without the necessary Modrinth metadata (such as those installed from CurseForge) will be placed as JARs into the modpack zip; make sure that you have the licenses for these mods [as it is your responsibility as a pack creator to](https://support.modrinth.com/en/articles/8797527-obtaining-modpack-permissions).
```

Comment:
The Modrinth pack format does support side-only mods! You only need to export a pack once and the launcher will filter out server-only mods, and you can use a tool like [mrpack-install](https://support.modrinth.com/en/articles/8802250-modpacks-on-modrinth) to install the mrpack file on a server.

Reply:
Thanks for clarifying that! I've updated the description to better reflect that.